Source/Elasticurl/ElasticurlOptions.swift

```diff
@@ -1,10 +1,14 @@
-//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-//  SPDX-License-Identifier: Apache-2.0.
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
 
 import AwsCommonRuntimeKit
 
-//swiftlint:disable identifier_name
-public struct ElasticurlOptions {
+// swiftlint:disable identifier_name
+public enum ElasticurlOptions {
```

Comment:
if static let are still as they are, there is no point converting to enum.

This is more like obj-c pattern.